components/scripts/lib/help.sh

```diff
@@ -51,6 +51,9 @@ print_option_usage() {
     -v)
        _print_option_usage "-v, --version" "Prints version info."
        ;;
+    -x)
+       _print_option_usage "-x, --fail-if-not-optimized" "Causes the script to exit with exit code -2 if the script detects that the build is not optimized."
+       ;;
```

Comment:
I would use the following user-facing values/text:

-f
--fail-if-not-fully-cacheable
"Terminates with exit code -2 if the build is not fully cacheable."
FAILURE: Build is not fully cacheable for the given task graph. (with no line break after)

Reply:
✅ Done